Chore: Update TestData README (#77607)
grafana/grafana: docs/sources/datasources/testdata/_index.md

[panels-visualizations]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations"
[panels-visualizations]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations"
{{% /docs/reference %}}

## Use a custom version of TestData

> If you want to use a version of TestData different from the one shipped with Grafana. Note that this procedure is still experimental (requires Grafana 10.3.0). You can do so by following these steps:

Reviewer: ```suggestion
{{% admonition type="note" %}}
This feature is experimental and requires Grafana version 10.3.0 as a minimum.
{{% /admonition %}}

If you want to use a version of TestData different from the one shipped with Grafana, you can do so by following these steps:
```

[panels-visualizations]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations"
[panels-visualizations]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations"
{{% /docs/reference %}}

## Use a custom version of TestData

{{% admonition type="note" %}}
This feature is experimental and requires Grafana version 10.3.0 as a minimum.
{{% /admonition %}}

If you want to use a version of TestData different from the one shipped with Grafana, you can do so by following these steps:

1. Enable the [feature toggle](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/) `externalCorePlugins`.

> 1. Set the configuration field `plugin.grafana-testdata-datasource.as_external` to `true`.

Reviewer 2: I wonder is this explicit enough for people to know what to put in the `ini`? Just not seen it before described this way

Author: fair point :+1:

[panels-visualizations]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations"
[panels-visualizations]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations"
{{% /docs/reference %}}

## Use a custom version of TestData

{{% admonition type="note" %}}

> This feature is experimental and requires Grafana version 10.3.0 as a minimum.

Reviewer 3: ```suggestion
This feature is experimental and requires Grafana version 10.3.0 or later.
```

{{% /admonition %}}

> If you want to use a version of TestData different from the one shipped with Grafana, you can do so by following these steps:

Reviewer 3: ```suggestion
If you want to use a version of TestData different from the one shipped with Grafana, follow these steps:
```

1. Enable the [feature toggle](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/) `externalCorePlugins`.
1. Set the configuration field `as_external` for the plugin to `true`. An example configuration would be:

   ```ini
   [feature_toggles]
   externalCorePlugins = true

   [plugin.grafana-testdata-datasource]
   as_external = true
   ```

1. Restart Grafana.

> These settings, if enabled, allows you to to install TestData as an external plugin and manage its lifecycle independently of Grafana.

Reviewer 3: ```suggestion
These settings, if enabled, allow you to to install TestData as an external plugin and manage its lifecycle independently of Grafana.
```
